test/column/editor.test.tsx

```diff
@@ -13,64 +13,64 @@ interface Row {
 }
 
 describe('Editor', () => {
-  it('should open editor on double click', () => {
+  it('should open editor on double click', async () => {
     render(<EditorTest />);
-    userEvent.click(getCellsAtRowIndex(0)[0]);
+    await userEvent.click(getCellsAtRowIndex(0)[0]);
     expect(screen.queryByLabelText('col1-editor')).not.toBeInTheDocument();
-    userEvent.dblClick(getCellsAtRowIndex(0)[0]);
+    await userEvent.dblClick(getCellsAtRowIndex(0)[0]);
     expect(screen.getByLabelText('col1-editor')).toHaveValue(1);
-    userEvent.type(document.activeElement!, '2');
-    userEvent.tab();
+    await userEvent.keyboard('2');
+    await userEvent.tab();
     expect(screen.queryByLabelText('col1-editor')).not.toBeInTheDocument();
-    expect(getCellsAtRowIndex(0)[0]).toHaveTextContent(/^12$/);
+    expect(getCellsAtRowIndex(0)[0]).toHaveTextContent(/^21$/);
```

Comment:
Try loading
```
data:text/html,<!doctype html><input autofocus value="0">
```
in a new tab, `autofocus` sets the selection range to the beginning of the input.
Using `.type` types the value at the end of the input rather than where the caret currently is.
I guess the behavior for `.keyboard` changed.